site/_apps/react-hn-kristofer.txt

```diff
@@ -15,11 +15,11 @@ hosting: 'Webfaction + Cloudflare'
 authors: 
     - name: 'Kristofer'
 lighthouse: '91/100'
-interactive-em: '2.68s'
-interactive-faster-3g: '2.25s'
-lighthouse-link: https://www.webpagetest.org/lighthouse.php?test=170919_KT_914eaf8bd26c8743187e1167d669c56e&run=3
-wpt-em-link: https://www.webpagetest.org/result/170919_A1_36750d6e522b0533185b268ddef79114/
-wpt-faster-3g-link: https://www.webpagetest.org/result/170919_KT_914eaf8bd26c8743187e1167d669c56e/
+interactive-em: '2.57s'
+interactive-faster-3g: '2.09s'
```

Comment:
Changes here LGTM. 

Do you have a feel for how much you would attribute the switch to React 16 RC to the improvements here, @kristoferbaxter ? I've been talking to Dan Abramov about the smaller vendor size this release brings and potential for some minor wins in start-up time.

Reply:
I'd attribute most of the improvement to React 16 RC. Other changes were minimal.

Reply:
That's excellent. We're seeing some other improvements to TTCI when switching over to 16 in other apps. Appreciate you testing the upgrade!